index.html

```diff
@@ -9025,6 +9039,267 @@ <h3><dfn>Take Element Screenshot</dfn></h3>
 </section> <!-- /Take Element Screenshot -->
 </section> <!-- /Screen capture -->
 
+<section>
+<h2>Print</h2>
+
+<p>The print functions are a mechanism to render the document to a
```

Comment:
Does this have anything to do with print media CSS? Or is the HTML rendering used?

Reply:
I would assume that print stylesheets are applied in the same way they would for actual printing, maybe that needs to be called out explicitly.